TrackPropagation/RungeKutta/src/RKCartesianDerivative.h

```diff
@@ -1,7 +1,7 @@
 #ifndef RKCartesianDerivative_H
 #define RKCartesianDerivative_H
 
-class CartesianState;
+#include "CartesianState.h"
```

Comment:
Better to include headers with the full path

Reply:
not for local header

Reply:
I agree: local headers (not in interface/) should be included like this.
And whoever is including A/B/src/B.h from a package C/D is doing it wrong.